packages/types/src/span.ts

```diff
@@ -21,13 +22,15 @@ export type SpanAttributeValue =
   | boolean
   | Array<null | undefined | string>
   | Array<null | undefined | number>
-  | Array<null | undefined | boolean>;
+  | Array<null | undefined | boolean>
+  | Measurements;
```

Comment:
This breaks OTEL compatibility unfortunately. We'll just add them to span data directly for now, but we need a bigger strategy afterwards.

Reply:
Yeah I think we'll probably have to flatten measurements into the attributes, or something similar in the future